### ISSUE TITLE
`-m` with ipdb?

### ISSUE THREAD

**Issue by MrMino** [gotcha/ipdb#192]
I don't know whether this request should be pointed at pdb rather than ipdb, ~but since chances of having this feature implemented any time soon by python core devs is probably close to nil,~ here goes:

Taken from `man python`:

```
      -m module-name
             Searches sys.path for the named module and runs the corresponding .py file as a script.
```

This can be used to run unittests:
```
python -m unittest discover
```

Sometimes when tests fail, in order to run a post mortem on them, I have to manually find the failing testcase, add `import ipdb; ipdb.set_trace()` to it, rerun the thing, <kbd>c</kbd>, <kbd>Enter</kbd>... only to find out that I added the `set_trace()` in the wrong line.

I'd like to kindly request that the `-m` be added to ipdb with the same semantics as the interpreter option.

This will make it very simple to debug testcases:

```
ipdb -m unittest discover
```
<kbd>c</kbd>, <kbd>Enter</kbd>
Traceback...
<kbd>d</kbd><kbd>d</kbd><kbd>d</kbd><kbd>d</kbd><kbd>d</kbd>... until the bottom frame
*profit*

**Comment by MrMino** [comment]
> but since chances of having this feature implemented any time soon by python core devs is probably close to nil, here goes:

Scratch that. https://docs.python.org/3/library/pdb.html:

> New in version 3.7: pdb.py now accepts a -m option that execute modules similar to the way python3 -m does. As with a script, the debugger will pause execution just before the first line of the module.

Turns out its actually only ipdb that is behind here.

**Comment by MrMino** [comment]
Hey, @gotcha, pretty please :)? I know you're probably seriously busy, and your todo-list is as long as your arm and a - understandable.
This is a simple, 20 line merge request which was done by analogy, nothing fancy there, and it adds a really cool feature. I'd love to use it on machines at work where we install ipdb automatically, without having to resort to patching it up every time.

**Comment by MrMino** [comment]
This has been released in v0.13.4, see linked PR.
